solved_exercises/dani/04-functions.py

```diff
@@ -0,0 +1,63 @@
+print("Calculate the factorial of a number (positive integer).")
+def factorial (n):
+    if n > 0:
+        result = n * factorial(n-1)
+    else:
+        result = 1
+    return result
+value = int(input("Please enter a numeric value:\n"))
+print("Factorial of",value,"is",factorial(value))
+
+print("Write a function that flattens a list.")
+def flat (lista):
+	return [val for sublist in lista for val in sublist]
+given_list = [[18.0], [13.8], [16.2], [56.5], [147], [18.2]]
+print (flat(given_list))
+
+print("Generate a list with Fibonacci sequence up to a number.")
+from math import sqrt
+def fibonacci (n):
+    result = []
+    for i in range(n):
+        if is_fibonaci(i) != None:
+            result.append(i)
+            if i == 1:
+                result.append(i)
+    return result
+
+def is_fibonaci (n):
+    formula1=5 * n**2 + 4
+    formula2=5 * n**2 - 4
+    if isPerfectSquare(formula1) or isPerfectSquare(formula2):
+        return n
```

Comment:
the function name suggest you would return a boolean, instead you return the same value that was passed in.

Reply:
I forgot about it, shall be True there